libs/agno/agno/workflow/workflow.py

```diff
@@ -2819,6 +2837,29 @@ def _run_workflow_agent_stream(
                     # workflow_agent field is used by consumers of the events to distinguish between workflow agent and regular agent
                     event.workflow_agent = True  # type: ignore
                 yield event  # type: ignore[misc]
+            elif isinstance(
```

Comment:
Shouldn't we just check for all events? I.e. `RunOutputEvents` and `TeamRunOutputEvents`?